crates/sui-kvstore/src/bigtable/client.rs

```diff
@@ -197,6 +200,29 @@ impl KeyValueStoreReader for BigTableClient {
         Ok(result)
     }
 
+    async fn get_checkpoint_summaries(
```

Comment:
Do we need this method for anything other than providing watermark info? If not, should we just merge them and keep a single one

Reply:
I ended up needing to keep a separate `get_latest_checkpoint_summary` and a `get_latest_checkpoint` because the latter is used by the `ProgressStore` impl.